src/components/complete-profile-fifth/CompleteProfileFifth.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { memo } from 'react';
+import { Slider, FilledInput, InputLabel, FormControl } from '@mui/material';
+import { useForm, Controller } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { useLocales } from 'src/locales';
+
+import { MARKS } from './constants';
+import { CompleteProfileFifthValues, useCompleteProfileFifthSchema } from './validation';
+import {
+  Wrapper,
+  StyledForm,
+  NextButton,
+  ButtonWrapper,
+  ErrorMessage,
+  InputWrapper,
+} from './styles';
+
+interface IProps {
+  onNext: () => void;
+}
+
+function CompleteProfileFifth({ onNext }: IProps): JSX.Element {
+  const { translate } = useLocales();
+  const completeProfileFifthSchema = useCompleteProfileFifthSchema();
+
+  const {
+    control,
+    handleSubmit,
+    formState: { isDirty, errors, isValid },
+  } = useForm<CompleteProfileFifthValues>({
+    mode: 'onBlur',
+    resolver: yupResolver(completeProfileFifthSchema),
+    defaultValues: {
+      rate: 0,
+    },
+  });
+
+  const onSubmit = handleSubmit(() => {
+    onNext();
+  });
+
+  return (
+    <Wrapper>
+      <StyledForm onSubmit={onSubmit}>
+        <Controller
+          name="rate"
+          control={control}
+          render={({ field }): JSX.Element => (
+            <Slider
+              {...field}
+              size="medium"
+              onChange={(value): void => field.onChange(value)}
+              aria-labelledby="input-slider"
+              min={0}
+              max={200}
```

Comment:
Would be nice if max value was took from constants

Reply:
thanks, fixed